pkg/simple/client/events/options.go

```diff
@@ -54,6 +57,21 @@ func (s *Options) AddFlags(fs *pflag.FlagSet, c *Options) {
 		"if this filed left blank, KubeSphere will use kubernetes builtin event API instead, and"+
 		" the following elastic search options will be ignored.")
 
+	fs.BoolVar(&s.BasicAuth, "events-elasticsearch-basicAuth", c.BasicAuth, ""+
+		"Elasticsearch service basic auth enabled. KubeSphere is using elastic as event store, "+
+		"if this filed left blank, KubeSphere will use kubernetes builtin log API instead, and"+
```

Comment:
Please follow the comments above and change this description as well.

Reply:
Ok